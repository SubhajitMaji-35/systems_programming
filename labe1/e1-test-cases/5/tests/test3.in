Kabul
Yerevan
Baku
Tbilisi
Tashkent
Astana
Bishkek
Dushanbe
Islamabad
New Delhi
Dhaka
Kathmandu
Thimphu
Colombo
