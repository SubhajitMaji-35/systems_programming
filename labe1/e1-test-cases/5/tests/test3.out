abl
ervan
ak
biisi
askent
stna
iskek
usanbe
slmaba
ewDelh
haa
atmand
hiphu
olmbo
